slurmci-test.toml

```diff
@@ -0,0 +1,76 @@
+# Tests to be run by SLURM CI
```

Comment:
Put your changes you made to this file in: [`.buildkite/pipeline.yml`](https://github.com/CliMA/ClimateMachine.jl/blob/4d41ba13eec1235119cf2679b1d2f43d911fe640/.buildkite/pipeline.yml)

Reply:
ok! in the previous slurm file, there was a cpu or gpu section. Would I just list file in both sections now?

Reply:
> Would I just list file in both sections now?

I believe so. Slightly unfortunate, but that's the way the system works.

Reply:
ok! no worries. just wanted to make sure. thanks!